### ISSUE TITLE
exporting an image to base64 without specifying a path

### ISSUE THREAD

**Issue by OmlineEditor** [helloworld1/FreeOTPPlus#137]
now when I export to json format, the picture for the account is indicated as:
file://data/user/0/folder.app/file.jpg
if I transfer this json file from the phone to my tablet or to the second phone, then I will not be able to download the pictures.
if i uninstall and reinstall the program i will lose all pictures. inside the json file, there is only the path to the picture, but the picture itself is not. I ask you to make it possible to export pictures and save pictures in base64 inside a json file.

**Comment by helloworld1** [comment]
Agreed. This is not the right way of exporting image data. The exported json file can end up quite big if images are included.

**Comment by OmlineEditor** [comment]
let the file be big that's not a problem. the problem is if there is no necessary icon or the icon will be lost completely if it is not saved in json.
